Update 'welcome' infobox on the homepage (#263)
pulsar-edit/pulsar-edit.github.io: docs/index.md

Welcome to all new visitors!

### Downloads and Releases

> We are constantly working on the editor and maintain both a rolling release and regular release model for our binaries.

Reviewer: ```suggestion
We are constantly working on the editor and maintain both rolling and regular releases.
```

Author: Fantastic, I wasn't fond of the original wording but couldn't quite decide how to change it. This has it just right, I'd say.

> You can find out more about the difference and download a binary of Pulsar by following [this link](https://pulsar-edit.dev/download.html)

Reviewer: ```suggestion
You can find out more about the differences and download Pulsar on our [downloads page](https://pulsar-edit.dev/download.html)
```
I know it isn't part of what you edited but I'm thinking we should change this (and the line above). I think references to binaries are confusing and not really necessary. It also makes it more generic so that people can find repo install links like deb-get and chocolatey.

Also changes the link text, I believe it is generally considered bad form for accessibility to just say "click here" etc.

Author: Agreed on not referencing "binaries" so heavily.

> Also changes the link text, I believe it is generally considered bad form for accessibility to just say "click here" etc.

Quite right, it is better for accessibility. (Besides SEO, I think, to whatever extent that matters. Hopefully none, but it might make bots understand relationships between our pages better somehow? And yeah, that's the point, relationships being clearer is the entire idea re: accessibility.)

and following the instructions.

> There are some community-maintained Pulsar packages in various formats, such as Flatpak, AUR, etc. The effort put into maintaining these packages is very much appreciated!

Reviewer: ```suggestion
There are some community-maintained packaged releases of Pulsar in various formats, such as Flatpak, AUR, etc. The effort put into maintaining these packages is very much appreciated!
```
Purely optional (and very happy for further suggestions) - yes the suggestion is a lot more unweildy but it makes it distinct from "Pulsar packages" i.e. the editor plugins.

Author: The whole paragraph might need a little second-time-over regarding "package" this and "package" that.

A good start, though.

Reviewer: Agree it is a bit hard going, probably does need a bit more rework.

Author: I have a gut feeling the answer is probably "say less" in this paragraph. So I think a trim will do it good once I have fresh eyes, or if enyone else wants to take a crack at just not saying as much, or staying to just. some facts and light remarks rather than being comprehensive, maybe. ASince it's just an infobox, after all.

Hey, maybe the cure is to get more into the weeds on Downloads in terms of _acknowledging_ these other formats, even if we don't tier-1 support them (and even if we say we don't totally support them), and hint that users interested in these can see more on the _Downloads page_?? Seems like a a good idea to me.

That said, the core Pulsar team doesn’t have the person-hours or familiarity with the technology to be deeply involved with these efforts.
The Pulsar Flatpak in particular has some minor known issues due to its sandboxing, but should still work overall.
The core team will make some effort to support users on these packaged versions of Pulsar and answer their questions, but we have less control over these packages/environments,
and less time to dedicate to them, so resolving issues may take more time and involve the community packaging maintainers, not just the core Pulsar team.
For significant technical barriers like Flatpak sandboxing, related issues may have no estimated time of completion for a fix.
Users are welcome to choose the packaging format they prefer, but your mileage may vary.

### App Updates

Currently Pulsar does not support automatic app updates. There are, however, automated notifications of new versions,

> which include links so you can easily get the new versions when they come out. (These notifications can be disabled if you wish.)

Reviewer: ```suggestion
which include links and instructions so you can easily get the new versions when they come out. (These notifications can be disabled if you wish.)
```
`pulsar-update` includes recognition of repo based installs as well (deb-get, flatpak, chocolatey) and shows instructions of what command to run.

Beyond that, new versions can be obtained via the Download links here on our website, or from our CI (continuous integration) pages as explained on our Download page.

### Blog

> _For updates on the goings-on around here,_ please do check out our [blog](https://pulsar-edit.dev/blog/)!

Reviewer: Not sure I get why this has been emphasised?

Author: It was originally sort of a joke on a heading simply labeled "Updates", but I changed that heading to "App Updates" and the disclaimer about community packaged versions of Pulsar is a bit long, so the connection really isn't too obvious now, I think. Oh well.

Emphasis could be removed, I suppose.

Reviewer: I thought it might be something like that, I was missing the link somewhat.

Author: I tweaked the emphasis to just be 
> For updates on the _goings-on_ around here, please do check out our [blog](https://pulsar-edit.dev/blog/)!
